packages/ilios-common/addon/components/course/objective-list-item.js

```diff
@@ -50,6 +51,10 @@ export default class CourseObjectiveListItemComponent extends Component {
     return this.isManagingParents || this.isManagingDescriptors || this.isManagingTerms;
   }
 
+  get fadeTextIsExpanded() {
```

Comment:
I think I caused this confusion because I was talking about a getter that would read both a local and a passed in property and decide which to use, but I'm not sure if this `get` is necessary. Couldn't the template just read `this.fadTextExpanded` directly?

Reply:
You're totally right. No getter needed. Just passing the property now.